Code/Editor/EditorFramework/EditorApp/ExternalTools.cpp

```diff
@@ -18,24 +18,34 @@ ezString ezQtEditorApp::FindToolApplication(const char* szToolName)
 
   ezEditorPreferencesUser* pPref = ezPreferences::QueryPreferences<ezEditorPreferencesUser>();
 
-  bool bFolders[2] = {false, true};
+  ezHybridArray<ezString, 3> sFolders;
+  sFolders.PushBack(ezApplicationServices::GetSingleton()->GetPrecompiledToolsFolder(false));
+  sFolders.PushBack(ezApplicationServices::GetSingleton()->GetPrecompiledToolsFolder(true));
 
   if (pPref->m_bUsePrecompiledTools)
   {
-    ezMath::Swap(bFolders[0], bFolders[1]);
+    if(!pPref->m_sCustomPrecompiledToolsFolder.IsEmpty() && ezOSFile::ExistsDirectory(pPref->m_sCustomPrecompiledToolsFolder))
+    {
+      ezStringBuilder customToolsFolder = pPref->m_sCustomPrecompiledToolsFolder;
+      customToolsFolder.MakeCleanPath();
+      sFolders.PushBack(customToolsFolder);
+      ezMath::Swap(sFolders[0], sFolders[2]);
```

Comment:
Wouldn't it make more sense if the order was custom, true, false in this case?

Reply:
This is exactly what the implementation currently does.